tests/pytests/common.py

```diff
@@ -657,13 +657,8 @@ def get_passphrase():
 # Use FT.* command to make sure that the module is loaded and initialized
 def verify_shard_init(shard):
     with TimeLimit(5, 'Failed to verify shard initialization'):
-        while True:
-            try:
-                shard.execute_command('FT.SEARCH', 'non-existing', '*')
-                raise Exception('Expected FT.SEARCH to fail')
-            except redis_exceptions.ResponseError as e:
-                if 'no such index' in str(e):
-                    break
+        shard.execute_command('FT.CREATE', 'idx' ,'SCHEMA', 't', 'TEXT')
```

Comment:
Consider giving it a less common name?
Also, doesn't it fail before the shard is initialized with "cannot distribute command"? That's why it was wrapped with a `try..except` before

Reply:
It's called before any indexes are created, but switched just in case.
Regarding the failure - doesn't seem to fail. Isn't the idea here to make sure that you don't get that error and the cluster is initialized appropriately?
The `try..except` before was catching the error that you expected upon indexing a non-existing index.

Reply:
there is also the `ERRCLUSTER Uninitialized cluster state, could not perform command` that may be returned if we call a search command before any `SEARCH.CLUSTERSET` or the periodic `SEARCH.CLUSTERREFRESH` commands were executed (and we don't know the topology yet)

Reply:
Added the wrapping `try..except` to catch this flakiness